onebusaway-android/src/main/res/values-w820dp/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
+         (such as screen margins) for screens with more than 820dp of available width. This
+         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```

Comment:
Looks like you added this in the initial commit for 25-tripPlan (7293e443).
Maybe it was added automagically by Android Studio for some reason.

On Mon, Jul 18, 2016 at 9:49 PM, Sean Barbeau <notifications@github.com>
wrote:

> In onebusaway-android/src/main/res/values-w820dp/dimens.xml
> <https://github.com/OneBusAway/onebusaway-android/pull/602#discussion_r71261413>
> :
>
> > @@ -0,0 +1,6 @@
> > +<resources>
> > +    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
> > +         (such as screen margins) for screens with more than 820dp of available width. This
> > +         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
> > +    <dimen name="activity_horizontal_margin">64dp</dimen>
>
> Just curious - why was this added for w820dp devices?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OneBusAway/onebusaway-android/pull/602/files/2d4af4db8bfb76b9dbeb2530345e90c96b338e01#r71261413>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFfDc-5T1zWTfS-DwtJNTYXypK18Mi7Kks5qXC0hgaJpZM4JPQ0k>
> .
>